data/update.py

```diff
@@ -47,6 +46,11 @@
 SCANNERS = os.environ.get("SCANNERS", "inspect,tls,analytics")
 ANALYTICS_URL = "https://analytics.usa.gov/data/live/second-level-domains.csv"
 
+# TODO:
+# --date: override date
+# --skip-scan: skip the scanning part (rely on local scan data)
+# --skip-upload: don't bother uploading anything to S3
```

Comment:
TODO above: add a command line options parser and support some helpful dev/debugging flags.